widgets/cta/styles/default.less

```diff
@@ -1,12 +1,25 @@
 
 @border_color: #E3E3E3;
 @background_color: #F8F8F8;
+@title_color: '';
```

Comment:
The widgets bundle supports variables of the form `@title_color: default`. This removes any attributes that are still at default. So they'll just use whatever the theme specifies.

You can get a better idea of how we use this in the Headline widget https://github.com/siteorigin/so-widgets-bundle/tree/develop/widgets/headline